roles/common/tasks/system-tools.yml

```diff
@@ -29,6 +29,22 @@
     - cdpr
     - sysstat
     - logrotate
+    - ipmitool
+    - ntp
```

Comment:
Moving ntp package here, which is /after/ the ntp.yml file is processed creates a problem. A crash because the ntp conf file doesn't exist. You need to move the ntp.yml include after system-tools.